packages/frontend/src/hooks/fungibleTokensIncludingNEAR.js

```diff
@@ -19,7 +19,7 @@ const fungibleTokensIncludingNEAR = (tokens, balance, nearTokenFiatValueUSD) =>
 export const useFungibleTokensIncludingNEAR = function () {
     const balance = useSelector(selectBalance);
     const accountId = useSelector(selectAccountId);
-    const tokens = useSelector((state) => selectTokensWithMetadataForAccountId(state, { accountId }));
+    const tokens = useSelector((state) => selectTokensWithMetadataForAccountId(state, { accountId })).sort((a, b) => a.name.localeCompare(b.name));
```

Comment:
Ideally this sorting should happen _inside the selector_, since they are memoized  (and it would be consistent with how NFT tokens are selected, see https://github.com/near/near-wallet/blob/4b6446273588841e0a4a923a8890accb6dfe6b82/packages/frontend/src/redux/slices/nft/index.js#L205-L224 )

Reply:
Makes sense - moved it 👍